packages/react/src/accordion/__snapshots__/index.test.tsx.snap

```diff
@@ -3,24 +3,49 @@
 exports[`CdsAccordion snapshot 1`] = `
 <div>
   <CdsAccordion>
-    <cds-accordion>
-      <CdsAccordionPanel
-        expanded={true}
+    <cds-accordion(CustomElement)
+      __forwardedRef={null}
+    >
+      <cds-accordion
+        __forwardedRef={null}
```

Comment:
The snapshots are updated due to a middle layer created from the lit wrapper fixing the ref issue.